tests/rules/javascript/test.eslintRulesObject.js

```diff
@@ -3,25 +3,24 @@ import { existsSync, readdirSync } from 'fs';
 
 import { ESLINT_RULE_MAPPING } from 'const';
 import * as jsRules from 'rules/javascript';
-import { ignorePrivateFunctions, singleLineString } from 'utils';
+import { ignorePrivateFunctions } from 'utils';
 
 
 describe('Eslint rules object', () => {
   it('should have keys that match the file names', () => {
     for (let jsRule in ignorePrivateFunctions(jsRules)) {
       var jsFilePath = path.join('src/rules/javascript/', `${jsRule}.js`);
-      assert.ok(existsSync(jsFilePath),
-                `key "${jsRule}" doesn't exist at "${jsFilePath}"`);
+      expect(existsSync(jsFilePath)).toBeTruthy();
     }
   });
 
   it('should have files that match the keys', () => {
     var files = readdirSync('src/rules/javascript')
       .filter((fileName) => fileName !== 'index.js');
     for (let fileName of files) {
-      assert.ok(ESLINT_RULE_MAPPING.hasOwnProperty(path.parse(fileName).name),
-                singleLineString`fileName "${fileName}" does not have a
-                matching key in the eslint rules object.`);
+      expect(
+        ESLINT_RULE_MAPPING.hasOwnProperty(path.parse(fileName).name)
+      ).toBeTruthy();
```

Comment:
I think it's worth look to use `.toBe(true)` more in cases like this and to avoid truthy/falsey comparisons where possible (I know this is a result of the code-mod) but it's good to not have potentially problematic loose comparisions.